types/src/side_effect.rs

```diff
@@ -32,9 +32,9 @@ pub const DATA_SIDE_EFFECT_ID: &[u8; 4] = b"data";
 pub struct SideEffect<AccountId, BlockNumber, BalanceOf> {
```

Comment:
You'll need to manually bump the version in the manifest here, also manually publish